fuse_variables/include/fuse_variables/orientation_2d_stamped.h

```diff
@@ -101,11 +113,15 @@ class Orientation2DLocalParameterization : public fuse_core::LocalParameterizati
     return true;
   }
 
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+  bool MinusJacobian(
+#else
   bool ComputeMinusJacobian(
+#endif
     const double* /*x*/,
     double* jacobian) const override
   {
-    jacobian[0] = 1.0;
+    jacobian[0] = -1.0;
```

Comment:
@nlunscher-cpr This analytic jacobian was wrong. This change goes together with fix `fuse_core::AutoDiffLocalParameterization`, that with the correct jacobian computed becomes the same as the one in `ceres::AutoDiffManifold`